### PR TITLE
Fe availability and pricing

### PR DESCRIPTION
**Pull request by quddd**
https://user-images.githubusercontent.com/24212950/105921398-c9676980-6006-11eb-8691-8ebf8b205347.mp4

### CODE REVIEW

File: client/src/components/dashboard/EditProfileForm.js
Comment (quddd) [inline client/src/components/dashboard/EditProfileForm.js:15]: Didn't change file. I think prettier formatted it lol. Still works fine on my end

File: client/src/components/availability/SelectTimeForm.js
Comment (ruslan-akhm) [inline client/src/components/availability/SelectTimeForm.js:83]: selectedDate is being checked to not be empty, but time is not. I pressed delete button on keyboard when choosing time and it passed into db
avail2

**Comment by ruslan-akhm** [comment]
Noticed couple things here:

- Console throws bunch of errors and warnings
- I was able to set availability time the way "from" was later that "to" (e.g. from 7pm to 5pm, pls don't mind me having 24hr format on screenshot) as well as can choose past day to set availability there 
- I also was able to chose availability while logged as dog owner (I believe only sitters need to set their availability).
- In calendar chosen date is highlighted, but a number inside of highlite area is off to the bottom. I'd suggest making that area smaller or of a little different shape? 

avail

**Comment by dproc96** [comment]
I'll echo what Ruslan has said, I think it would also be good to have some indication of the times you've set as available. Either a list or some kind of highlight on the calendar to indicate you have availability on the day

**Review by dproc96 — APPROVED** [review]
I've tested this and the happy path is working for me! There's still some edge cases (like adding an availability with a start time after end time) but I think we can go ahead and merge this to unblock integration and then worry about the edge cases separately.

If the deleteAvailability controller isn't being used then you can delete it before merging

**Pull request merged**
